src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SentenceCalculation.kt

```diff
@@ -47,7 +47,13 @@ data class SentenceCalculation(
     val expiryDateType = if (releaseDateTypes.contains(ReleaseDateType.SLED)) "SLED" else "SED"
     val releaseDateType = releaseDateTypes.toString()
 
-    return "Date of $expiryDateType\t:\t${unadjustedExpiryDate.format(formatter)}\n" +
+    var append = ""
```

Comment:
Minor; you could perhaps do with an immutable`val` like this:
```
val append = (sentence is Sentence) ? "Identification Track\t:\t${(sentence as Sentence).identificationTrack}\n" : ""
```
Also, given how you are using it in the return statement, is it append or prepend ?